packages/react-scripts/scripts/eject.js

```diff
@@ -76,51 +80,55 @@ prompt(
 
   var ownPackage = require(path.join(ownPath, 'package.json'));
   var appPackage = require(path.join(appPath, 'package.json'));
-
+  console.log(cyan('Updating dependencies...'));
   var ownPackageName = ownPackage.name;
-  console.log('Removing dependency: ' + ownPackageName);
+  console.log('  Removing dependency: ' + cyan(ownPackageName));
   delete appPackage.devDependencies[ownPackageName];
 
   Object.keys(ownPackage.dependencies).forEach(function (key) {
     // For some reason optionalDependencies end up in dependencies after install
     if (ownPackage.optionalDependencies[key]) {
       return;
     }
-    console.log('Adding dependency: ' + key);
+    console.log('  Adding dependency: ' + cyan(key));
     appPackage.devDependencies[key] = ownPackage.dependencies[key];
   });
-
-  console.log('Updating scripts');
+  console.log();
+  console.log(cyan('Updating scripts...'));
   delete appPackage.scripts['eject'];
   Object.keys(appPackage.scripts).forEach(function (key) {
     appPackage.scripts[key] = appPackage.scripts[key]
       .replace(
         new RegExp(ownPackageName + ' (\\w+)', 'g'),
         'node scripts/$1.js'
       );
+    console.log('  Replacing react-scripts ' + cyan(key) + ' with ' + cyan(appPackage.scripts[key]));
```

Comment:
Please make it so `react-scripts` is also cyan here. We're replacing a complete command.
Also let's put both `"react-scripts start/test/build"` and `"node scripts/whatever.js"` in quotes.

```
  Replacing "react-scripts start" with "node scripts/start.js"
```